R/adjmat.r

```diff
@@ -462,31 +462,52 @@ adjmat_to_edgelist.list <- function(graph, undirected, keep.isolates) {
 #' @keywords manip
 #' @include graph_data.r
 #' @author George G. Vega Yon & Thomas W. Valente
-toa_mat <- function(obj, labels=NULL, t0=NULL, t1=NULL) {
+toa_mat <- function(obj, num_of_behaviors=1, labels=NULL, t0=NULL, t1=NULL) {
```

Comment:
The number of behaviors should be implicit (so you can figure that out from diffnet object). So don't ask the user of it.

Reply:
I just deleted the 'num_of_behaviors' dependency in toa_mat and other functions.